### ISSUE TITLE
MySQL installer does not install packages

### ISSUE THREAD

**Issue by mynameismon** [microsoft/winget-pkgs#22462]
Create issues related to the winget.exe client [here](https://github.com/microsoft/winget-cli/issues/new?template=Bug_Report.md)

# Brief description of your issue

While installing MySQL via winget, only the installer is installed, and not the packages that are supposed to accompany the installer
# Steps to reproduce
1. Run ``winget install Oracle.MySQL``
2. Navigate to the folder where the installer has been installed and run MySQLInstaller.exe

# Expected behavior
1. The installer should install normally, with the various packages installing as usual

# Actual behavior
1. The installer can not find the required packages for installation
![image](https://user-images.githubusercontent.com/42601474/127098113-0dde2715-ddb2-4f94-9425-3b36863c2088.png)

# Environment

```none
[winget --info]
Windows: Windows.Desktop v10.0.22000.100
Package: Microsoft.DesktopAppInstaller v1.15.11694.0
MySQL version: 8.0.25
```

**Comment by jedieaston** [comment]
I've figured it out, there's two issues:

- There's a (bug/confusing behavior)[https://bugs.mysql.com/bug.php?id=92600] in the MySQL Installer (for two years now) that if you launch it without command line arguments it assumes you want the commercial installer instead of the community one. This one is easy-ish for us to fix, we just need to add `CONSOLEARGS="install -type=Full -silent"` to the custom installer flags for it to work. Since you have it installed already, just run `C:\Program Files (x86)\MySQL\MySQL Installer for Windows\MySQLInstaller.exe Community` from a shell to get the installer to behave.

- The larger problem is that even if we add those arguments to the manifest, it still won't work right because it can't install the VC++ 2015-2019 redist without user interaction. If they are installed it works fine, if they aren't it blinks a command window up on the screen and exits. It works fine interactively since it can ask you to install them, but not silently.

TL;DR: There's a workaround, but in general this is probably blocked for dependencies right now.
